vespajlib/src/main/java/com/yahoo/concurrent/maintenance/Maintainer.java

```diff
@@ -43,7 +43,8 @@ public Maintainer(String name, Duration interval, Instant startedAt, JobControl
         this.ignoreCollision = ignoreCollision;
         Objects.requireNonNull(startedAt);
         Objects.requireNonNull(clusterHostnames);
-        Duration initialDelay = staggeredDelay(interval, startedAt, HostName.getLocalhost(), clusterHostnames);
+        Duration initialDelay = staggeredDelay(interval, startedAt, HostName.getLocalhost(), clusterHostnames)
+                                .plus(Duration.ofSeconds(30));
```

Comment:
Consider extracting a descriptive constant for this fixed delay.

Reply:
Added a description